cmd/neofs-cli/modules/netmap/netinfo.go

```diff
@@ -14,8 +14,9 @@ import (
 var netInfoCmd = &cobra.Command{
 	Use:   "netinfo",
 	Short: "Get information about NeoFS network",
-	Long:  "Get information about NeoFS network",
-	Args:  cobra.NoArgs,
+	Long: `Get information about NeoFS network. 
+Other parameters of NeoFS network configuration are stored as bytes and printed in HEX.`,
```

Comment:
> are stored as bytes

well, everything is stored as bytes mb something like "Unknown (non-system) network parameters are displayed in HEX encoding."

Reply:
"Unknown configuration settings are displayed in hexadecimal format"?

Reply:
ok to me too